sdcm/nemesis.py

```diff
@@ -3485,20 +3485,17 @@ def decommission_nodes(self, add_nodes_number, rack, is_seed: Optional[Union[boo
                           f'{str(exc)}').publish()
 
     def disrupt_grow_shrink_cluster(self):
-        self._grow_cluster(rack=0)
-        self._shrink_cluster(rack=0)
+        self._grow_cluster(rack=None)
+        self._shrink_cluster(rack=None)
```

Comment:
you changed only _grow_cluster function logic tu support None and convert it to ) but not _shrink_cluster
and i see that 
_shrink_cluster calls decommission_nodes, that calls set_target_node, that calls _get_target_nodes

and putting None instead of 0 - might  change test behaviour
```
        if rack is not None:
            nodes = [node for node in nodes if node.rack == rack]
```

please test it before merging,
otherwise, it is better just to change disrupt_run_unique_sequence function for commit "improvement(perf during ops): change operations order' :
self._grow_cluster(rack=None) -> self._grow_cluster(rack=0) and
self._shrink_cluster(rack=None) -> self._grow_cluster(rack=0)

Reply:
https://jenkins.scylladb.com/job/scylla-staging/job/fabio/job/perf-acceptance-criteria/job/fix-rack-order/2/

Reply:
the `_shrink_cluster` will run `decommission_nodes` in a loop, so it can't be using `self.target_node`, as the target node is only 1 (and here we will iterate 3 times in this for-loop).

changing the `rack` from `0` to `None` doesn't change anything, because in this test we don't use the concept of racks at all